### PR TITLE
yocto: Add optional params to fetch all

### PR DESCRIPTION
**Pull request by rpannek**
Right now it is impossible to pass additional parameters to bitbake
which would change what would be fetched during the fetch phase.
This change takes all aditioal arguments and passes them to bitbake,
this way one can for example add -c populate_sdk which would trigger
fetch for different components then without it.

This change is backwards compatible, it just looks for additional
parameters.

### CODE REVIEW

File: yocto/fetch-sources-for-recipes.sh [hunk 1]
```diff
@@ -27,6 +27,9 @@
 
 YOCTO_DIR="$1"
 IMAGES="$2"
+shift
+shift
```
Comment (martin-ejdestig) [inline yocto/fetch-sources-for-recipes.sh]: Could do `shift 2` here, I think.

**Comment by martin-ejdestig** [comment]
Nit: "aditioal" -> "additional" in two places of commit message.

File: yocto/fetch-sources-for-recipes.sh (hunk 1 above)
Comment (martin-ejdestig) [reply yocto/fetch-sources-for-recipes.sh]: I think I prefer "shift 2", but if you want, we can leave it as is as well. :)

Comment (rpannek) [reply yocto/fetch-sources-for-recipes.sh]: [x] fixed

**Pull request merged**